### DISCUSSION TITLE
Rationale around cwd vs root keymaps

### DISCUSSION THREAD

**Discussion by elijahmanor** [LazyVim/LazyVim#83] [General]
I was reading through all the keymaps in the new docs (which are amazing by the way) and noticed a slight discrepancy between what a lowercase and uppercase character meant with regards to a `cwd` versus a `root dir` scope.

The main outlier seems to be with Lazygit. Is there a reason it was setup differently than the rest? If so, I'd love to heard it. If not, maybe switch it to be consistent with the others?

Examples:

## Lowercase means `cwd` & Uppercase means `root dir`

```
<leader>gg | Lazygit (cwd)
<leader>gG | Lazygit (root dir)
```

## Lowercase means `root dir` & Uppercase means `cwd`

```
<leader>ft | Terminal (root dir)
<leader>fT | Terminal (cwd)
```

```
<leader>fe | Explorer NeoTree (root dir)
<leader>fE | Explorer NeoTree (cwd)
<leader>e | Explorer NeoTree (root dir)
<leader>E | Explorer NeoTree (cwd)
```

```
<leader>ff | Find Files (root dir)
<leader>fF | Find Files (cwd)
```

```
<leader>sg | telescope.nvim Grep (root dir)
<leader>sG | telescope.nvim Grep (cwd)
```

```
<leader>sw | telescope.nvim Word (root dir)
<leader>sW | telescope.nvim Word (cwd)
```

**Comment by folke** [comment]
That was actually on purpose, but indeed not ideal in terms of consistency.

Reason is mainly, that when I'm in one of my projects, and I do `gd` to a file in another project, then in 99% of the cases when I want to open lazygit, I want to open it in my main project (cwd) and not the project I'm currently looking at (Neovim for example).

On the other hand, it might still be better to make the keymaps consistent. Worst case I'll do `<leader>gg`, realize I'm in another project and do `<leader>gG` instead.

**Reply by elijahmanor** [reply]
Yeah, it's a bit tricky. I can see consistency on lowercase vs uppercase (in terms of cwd vs root), but it also makes sense to normalize lowercase to be the most frequently needed use-case vs an alternate less used variation for uppercase. 

With that said, knowing what is typical in this case could be hard. I can see numerous use-cases for the git keymaps.

For simple projects, I create a tmux instance for the directory it is located (where the git root is).

However, we also have a monorepo where I make a tmux instance that has lots of sub-projects, but there is only one git root at the top level.

But then others might launch Neovim in a github type folder that has all their work in it with numerous git roots as sub-folders. So yeah, a bit tricky to know what is common or not.

**Reply by folke** [reply]
I've changed it to be consistent with the other keymaps. Makes more sense and will lead to less surprises. Thank you for the suggestion!

**Reply by elijahmanor** [reply]
Thanks! Appreciate it
